### PR TITLE
fix: enlarge cards and grid gap

### PR DESCRIPTION
**Pull request by exu3**
addressed styling issues with card size and spacing between cards

fixes Styling issues with Variant 1 of Card #26

### CODE REVIEW

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
still doesn't spam entire screen, gap too big on mobile, change the gap depending on device size

File: pages/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default function Home() {
                         d="M0 64h24c24 0 72 0 120 5.3 48 5.7 96 15.7 144 37.4 48 21.3 96 53.3 144 80 48 26.3 96 48.3 144 64C624 267 672 277 720 272s96-27 144-42.7c48-16.3 96-26.3 144-32 48-5.3 96-5.3 144 16 48 21.7 96 63.7 144 69.4 48 5.3 96-26.7 120-42.7l24-16v96H0z"
                     />
                 </svg>
-                <div className="justify-center grid sm:grid-flow-row md:grid-flow-col gap-6">
+                <div className="justify-center grid sm:grid-flow-row lg:grid-flow-col gap-12">
```
Comment (sarthaktexas) [inline pages/index.js]: if this isn't the fanciest thing i've ever seen, just do a manual 3 column layout lol and then change it to 2 on sm and above and then 1 on mobile

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

**Comment by awlevin** [comment]
#### 📝Changed routes:


| [**/**](https://open-o9b7q1016-thinkfiveable.vercel.app/) |
|:-:|
|<a href="https://open-o9b7q1016-thinkfiveable.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/p8SgO9zug7uaXePMi2x0b4U9Rdw_.png">(view full size)</a> |







Commit 224bd9a3b4cf3fc0dca53593fa5af0dd57981157 (https://open-o9b7q1016-thinkfiveable.vercel.app).

File: pages/index.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ export default function Home() {
                         d="M0 64h24c24 0 72 0 120 5.3 48 5.7 96 15.7 144 37.4 48 21.3 96 53.3 144 80 48 26.3 96 48.3 144 64C624 267 672 277 720 272s96-27 144-42.7c48-16.3 96-26.3 144-32 48-5.3 96-5.3 144 16 48 21.7 96 63.7 144 69.4 48 5.3 96-26.7 120-42.7l24-16v96H0z"
                     />
                 </svg>
-                <div className="justify-center grid sm:grid-flow-row md:grid-flow-col gap-6">
+                <div className="mx-8 justify-center grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-12">
```
Comment (sarthaktexas) [inline pages/index.js]: are you trying to center it? if so, just do:
```suggestion
                <div className="mx-auto justify-center grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-12">
```

**Review by sarthaktexas — APPROVED** [review]

**Comment by awlevin** [comment]
Wait what's this "changed routes" comment i left.....? I _swear_ that wasn't me

**Comment by sarthaktexas** [comment]
@awlevin it's called vercel commit preview, you enabled it. i got an email too if you want proof.

**Comment by awlevin** [comment]
lmao awkward. had a weird hunch that might’ve been what it was. when I enabled it earlier (mostly intended for a private repo) for some reason I thought it had more to do with preview deploy summaries (like build sizes etc.). 

I presume we can disable it on a per-project basis, and can look into that if we want... unless we like it? Curious what others think.

**Comment by sarthaktexas** [comment]
I like it cause I can see the desktop preview on mobile cause I review most of not a of the PRs (i'm typing this too) on mobile.

**Review by zaida04 — APPROVED** [review]

**Pull request merged**